app/src/main/java/com/verygoodsecurity/demoapp/VGSSampleApplication.kt

```diff
@@ -1,13 +1,5 @@
 package com.verygoodsecurity.demoapp
 
 import android.app.Application
-import com.microblink.blinkcard.MicroblinkSDK
 
-class VGSSampleApplication: Application() {
-
-    override fun onCreate() {
-        super.onCreate()
-//        MicroblinkSDK.setLicenseKey("<key>", this)
-//        MicroblinkSDK.setLicenseKey("<path>", this)
-    }
-}
\ No newline at end of file
+class VGSSampleApplication: Application()
```

Comment:
@DonaldRG I think this is used for MicroblinkSDK demo example, please suppress the lint check and leave it as is.

Reply:
Okis!